test/jdk/java/io/File/TempDirectoryNotExistingErrorMessage.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/* @test
+ * @bug 8290313
+ * @library /test/lib
+ * @summary Produce warning when user specified java.io.tmpdir directory doesn't exist
+ */
+
+import jdk.test.lib.process.ProcessTools;
+
+import java.io.File;
+
+public class TempDirectoryNotExistingErrorMessage {
```

Comment:
The tests look the same and maybe we can get away with one in java/io/File. Renaming it something like The TempDirectoryDoesNotExist would make it a bit more consistent with the tests in the Files test directory.